fdbserver/Ratekeeper.actor.cpp

```diff
@@ -1050,6 +1054,19 @@ void updateRate(RatekeeperData* self, RatekeeperLimits* limits) {
 			} else {
 				ssLimitReason = limitReason_t::storage_server_min_free_space_ratio;
 			}
+			if (printRateKeepLimitReasonDetails) {
+				TraceEvent("RatekeeperLimitReasonDetails")
+				    .detail("Reason", ssLimitReason)
+				    .detail("SSID", ss.id)
+				    .detail("SERVER_KNOBS->MIN_AVAILABLE_SPACE", SERVER_KNOBS->MIN_AVAILABLE_SPACE)
+				    .detail("SERVER_KNOBS->MIN_AVAILABLE_SPACE_RATIO", SERVER_KNOBS->MIN_AVAILABLE_SPACE_RATIO)
+				    .detail("ss.smoothTotalSpace.smoothTotal", ss.smoothTotalSpace.smoothTotal())
+				    .detail("ss.smoothFreeSpace.smoothTotal", ss.smoothFreeSpace.smoothTotal())
+				    .detail("targetBytes", targetBytes)
```

Comment:
1. field name needs to start with upper case. "targetBytes" -> "TargetBytes"
2. field name should not have special characters. such as "->" or ".". 

Please kick off a Joshua run, those special characters may cause test to fail.